gnc/sub8_thruster_mapper/nodes/mapper.py

```diff
@@ -48,13 +49,24 @@ def __init__(self):
         self.B = self.generate_B(self.thruster_layout)
         self.min_thrusts, self.max_thrusts = self.get_ranges()
 
+        #self.kill_listener = AlarmListener('kill', self.kill_cb)
+        # Not the proper way to do this at all, but the kill listener class is being wonky right now
+        self.kill_listener = rospy.Subscriber('/alarm_raise', Alarm, self.kill_cb)
```

Comment:
Uh-oh, what are the alarm errors? Looking at it, I see some rospy.logerr debugs that slipped through, but it runs dandily on my machine.